Documentation/kubernetes/policy.rst

```diff
@@ -34,13 +34,13 @@ For more information, see the official `NetworkPolicy documentation
 
 Known missing features for Kubernetes Network Policy:
 
-+----------------------------+----------------------------------------------+
-| Feature                    | Tracking Issue                               |
-+============================+==============================================+
-| Use of named ports         | https://github.com/cilium/cilium/issues/2942 |
-+----------------------------+----------------------------------------------+
-| CIDR-based L4 policy       | https://github.com/cilium/cilium/issues/1684 |
-+----------------------------+----------------------------------------------+
++------------------------------+----------------------------------------------+
+| Feature                      | Tracking Issue                               |
++==============================+==============================================+
+| Use of named ports           | https://github.com/cilium/cilium/issues/2942 |
++------------------------------+----------------------------------------------+
+| Ingress CIDR-based L4 policy | https://github.com/cilium/cilium/issues/1684 |
```

Comment:
This part shouldn't change, 1.0 doesn't have support for any CIDR-based L4 policy.

Reply:
Nothing changed here, only the feature name

Reply:
It's changed in master because master supports egress CIDR-based L4 Policy. That said, it doesn't matter much because if someone clicks through the links they should be able to figure out that the feature was merged in v1.1.